README.md

```diff
@@ -28,11 +27,26 @@ E or R, when already that color, resets it to black.
 There are also keybindings with Shift and Ctrl, which you can view in-app.
 Other neat things you can do:
 
-- Hit the save button to save to a PDF.
-  The export button exports to a JSON file, which you can later open and edit.
-- Hit Ctrl+V to paste images from the system clipboard.
-  You can also drag images to the board.
-- Right-clicking to bring up a context menu to change the style.
+- Hit the export button to save to a PDF.
+  The save button exports to a JSON file, which you can later load back in to qboard.
+- Use your browser's paste function (usually Ctrl+V or Cmd+V on Mac) to paste images from the system clipboard.
+  You can also drag images onto the board, or use the file picker on the left side of the screen (labeled `Open`).
+- Open your saved JSON files with the file picker to replace your current board with one from a file.
+  Alternatively, drag and drop your saved JSON files;
+  instead of overwriting your current board, this will insert the contents of the files into the board after the current page!
+- Right-click to bring up a context menu to change the style.
+- If you have a saved JSON file accessible via a URL, you can make a link that preloads the board with that file!
+Take any link to qboard, like <https://pihart.github.io/qboard/>, and append `?json=<your URL here>`, to get <https://pihart.github.io/qboard/?json=example.json>.
+
+The app protects you from losing your work.
```

Comment:
don't think this information is relevant; think we can cut these lines

Reply:
Assuming you're referring to the protecting work thing.

I'm fine with doing so (morally, i don't think it belongs in the readme either), however we received an issue based on misunderstanding (or at least not realizing) this, and thus i also put it as a task in the issue I'm resolving with this PR.

Reply:
yeah referring to that. i mean… it only would really come up by accident, right? and then it's a feature you'd just be glad is there? idk. i feel like almost everything these days has such a feature so i'm not really sure it's important enough to warrant a mention here

at this point there should be separate docs for not headline features xp

Reply:
Agreed, I'll remove this. And you're bringing up a point that's like what i'd said in the originating issue, locality of documentation is important. If we just write some basic plain-english documentation for these features in the code, we could surely just run typedoc or something on it.

Reply:
Removed in 7ebb4f7, though i think the text of the documentation is important for explaining the logic (otherwise the only way of knowing is looking for all of the `this.canvas.modified` occurrences, and i believe most wouldn't even think to do that).